FAR-Fullstack/PrimeTime
# Dates and nice calendar

How to test:
- yarn 
- go to add an event and see the beautiful calendar display. 
- home page and index: see the beautiful dates...
